code/modules/client/preference/preferences.dm

```diff
@@ -2035,6 +2038,10 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 	character.add_language(language)
 
+	if(species == "Grey")
+		if(speciesprefs)
+			character.speech_span = "wingdings"
```

Comment:
You cooould probably move the `speech_span` entirely to the mind, make the wingdings permanent regardless of what species you become. Once you go ☝︎✋︎👌︎👌︎☜︎☼︎✋︎💧︎☟︎, you never go back.